testcontainers/kafka.py

```diff
@@ -0,0 +1,73 @@
+import tarfile
+import time
+from io import BytesIO
+
+import kafka
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_container_is_ready
+
+
+class KafkaContainer(DockerContainer):
+    KAFKA_PORT = 9093
+    TC_START_SCRIPT = '/tc-start.sh'
+
+    def __init__(self, image="confluentinc/cp-kafka:5.4.3", port_to_expose=KAFKA_PORT):
+        super(KafkaContainer, self).__init__(image)
+        self.port_to_expose = port_to_expose
+        self.with_exposed_ports(self.port_to_expose)
+        self.with_env('KAFKA_LISTENERS', 'PLAINTEXT://0.0.0.0:9093,BROKER://0.0.0.0:9092')
+        self.with_env('KAFKA_LISTENER_SECURITY_PROTOCOL_MAP',
+                      'BROKER:PLAINTEXT,PLAINTEXT:PLAINTEXT')
+        self.with_env('KAFKA_INTER_BROKER_LISTENER_NAME', 'BROKER')
+
+        self.with_env('KAFKA_BROKER_ID', '1')
+        self.with_env('KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR', '1')
+        self.with_env('KAFKA_OFFSETS_TOPIC_NUM_PARTITIONS', '1')
+        self.with_env('KAFKA_LOG_FLUSH_INTERVAL_MESSAGES', '10000000')
+        self.with_env('KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS', '0')
+
+    def get_bootstrap_servers(self):
+        host = self.get_container_host_ip()
+        port = self.get_exposed_port(KafkaContainer.KAFKA_PORT)
+        return '{}:{}'.format(host, port)
+
+    @wait_container_is_ready()
+    def _connect(self):
+        bootstrap_servers = self.get_bootstrap_servers()
+        consumer = kafka.KafkaConsumer(group_id='test', bootstrap_servers=[bootstrap_servers])
+        topics = consumer.topics()
+        if not topics:
+            raise Exception()
```

Comment:
Can we raise a more specific exception here (including an error message)?

Reply:
Kafka library doesn't have any specific exception. But we should definitely add the helpful message here.  👍🏼

Reply:
I was thinking we should raise an exception here that is more specific than `Exception`, e.g. a `RuntimeError` or `ValueError` (or another one) depending on which is most appropriate.